app/assets/javascripts/listings/ListingUnitService.js.coffee

```diff
@@ -57,12 +57,20 @@ ListingUnitService = ($translate, $http, ListingConstantsService, ListingIdentit
       summaries.push(summary)
     summaries
 
-  Service._getAMIAmount = (incomeList, occupancy) ->
+  Service._getAnnualAMIAmount = (incomeList, occupancy) ->
     if incomeList && incomeList.values?
       _.find(incomeList['values'], {'numOfHousehold': occupancy})['amount']
     else
       console.error('Missing AMIs for occupancy', incomeList, occupancy)
 
+  Service._convertMaxAnnualToMonthly = (maxAnnualIncome) ->
```

Comment:
is `_` how we denote a service method that should only be used within the service?

Reply:
yep!